test/ssl_test_ctx.h

```diff
@@ -60,6 +60,11 @@ typedef enum {
     SSL_TEST_HANDSHAKE_RENEGOTIATE
 } ssl_handshake_mode_t;
 
+typedef enum {
```

Comment:
Well, apps does it one way and everything else (not just CT, also statem, ec etc) does it the other way. I can change it (everywhere in this file then) but... why?